pkg/browser/browser.go

```diff
@@ -30,7 +31,11 @@ func ForOS(goos, url string) *exec.Cmd {
 		r := strings.NewReplacer("&", "^&")
 		args = append(args, "/c", "start", r.Replace(url))
 	default:
-		exe = "xdg-open"
+		if inWsl() {
```

Comment:
This is cool; thanks for looking that up!

But I fear that in the long-term, filesystem-based OS detection might be brittle. That, and in the future, WSL might start shipping with `xdg-open`, so we'd want to use that if available.

How about simply falling back to `wslview` if it's available when `xdg-open` isn't?